oauth2client/transport/httplib2.py

```diff
@@ -12,22 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import absolute_import
```

Comment:
The past where `import httplib2` will try to import `oauth2client.transport.httplib2` instead of `httplib2`.